releasenotes/notes/prelude-0.8.1-d98e520584b5d73e.yaml

```diff
@@ -0,0 +1,8 @@
+---
+prelude: >
+    Qiskit Experiments 0.8.1 is a minor update to 0.8.0 to improve
+    compatibility with dependencies Qiskit and CVXPY. References to code in
+    these dependencies was updated to avoid triggering deprecation warnings, so
+    that it will continue to work as new versions of Qiskit and CVXPY are
```

Comment:
```suggestion
    compatibility with dependencies Qiskit and CVXPY. References to code in
    these dependencies were updated to avoid triggering deprecation warnings, so
    that the code will continue to work as new versions of Qiskit and CVXPY are
```